programs/MouseDemo.hs

```diff
@@ -51,6 +56,18 @@ buttonLayer st =
                padLeftRight (if wasClicked then 2 else 3) $
                str (if wasClicked then "<" <> label <> ">" else label)
 
+editLayer :: St -> Widget Name
+editLayer st =
+  B.border $
+  C.hCenterLayer $
+  vLimit 8 $
+  -- n.b. if clickable and viewport are inverted here, click event
+  -- coordinates will only identify the viewable range, not the actual
+  -- editor widget coordinates.
```

Comment:
The editor already embeds a viewport, so this use of `clickable` is still outside that viewport here. I would therefore expect any click events to still be just in terms of the viewed area, not the actual editor coordinates. Are you seeing different behavior than I'm describing? (In particular, I'd expect this to still report click events within the viewed area even if the editor was scrolled off to the right, say.)